src/test/java/com/puppycrawl/tools/checkstyle/internal/CommitValidationTest.java

```diff
@@ -94,7 +94,7 @@ public class CommitValidationTest {
 
     private static List<RevCommit> lastCommits;
 
-    @BeforeAll
+    @BeforeEach
     public static void setUp() throws Exception {
         lastCommits = getCommitsToCheck();
```

Comment:
This should not be a before each. This is basically a caching method and is not even part of the pitest suite.